ext/storage_sources/azure_store/azure_storage_source.cpp

```diff
@@ -98,6 +99,23 @@ static int azure_file_read(WT_FILE_HANDLE *, WT_SESSION *, wt_off_t, size_t, voi
   __attribute__((__unused__));
 static int azure_file_size(WT_FILE_HANDLE *, WT_SESSION *, wt_off_t *) __attribute__((__unused__));
 
+// Construct a pathname from the directory and the object name.
+static std::string
+azure_path(const std::string &dir, const std::string &name)
```

Comment:
hmm name here feels too ambiguous, it feels like name variable is more of a path, is that correct?

Reply:
i've removed this from the implementation